### PR TITLE
prepared release of version 2.0.7

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- release 2.0.7

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.mdx [hunk 1]
```diff
@@ -16,6 +16,14 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.0.7] - 2020-11-24
+                       
+
+### Bugfixes
+- [#5974](https://github.com/rasahq/rasa/issues/5974): `ActionRestart` will now trigger `ActionSessionStart` as a followup action.
+- [#7317](https://github.com/rasahq/rasa/issues/7317): Fixed Rasa Open Source not being able to fetch models from certain URLs.
```
Comment (akelad) [inline CHANGELOG.mdx:24]: @erohmensing can we elaborate here that this fixes an issue with Rasa X that was introduced in version 2.0.3?

Comment (erohmensing) [reply CHANGELOG.mdx:24]: Yes! Had a window open for this lol

Comment (akelad) [reply CHANGELOG.mdx:24]: lol thanks :D

Comment (erohmensing) [reply CHANGELOG.mdx:24]:
```suggestion
- [#7317](https://github.com/rasahq/rasa/issues/7317): Fixed Rasa Open Source not being able to fetch models from certain URLs.

  This addresses an issue introduced in 2.0.3 where `rasa-production` could not use the models from `rasa-x` in Rasa X server mode.
```

**Review by akelad — APPROVED** [review]

**Pull request merged**